benchmark/http/chunked.js

```diff
@@ -11,14 +11,14 @@
 var common = require('../common.js');
 
 var bench = common.createBenchmark(main, {
-  num: [1, 4, 8, 16],
-  size: [1, 64, 256],
+  n: [1, 4, 8, 16],
+  len: [1, 64, 256],
```

Comment:
Also if we want to do https://github.com/nodejs/node/issues/12068, doing this to other benchmarks too will make it eaiser..